cmd/openshift-install/gather.go

```diff
@@ -66,6 +67,7 @@ func newGatherBootstrapCmd() *cobra.Command {
 	}
 	cmd.PersistentFlags().StringVar(&gatherBootstrapOpts.bootstrap, "bootstrap", "", "Hostname or IP of the bootstrap host")
 	cmd.PersistentFlags().StringArrayVar(&gatherBootstrapOpts.masters, "master", []string{}, "Hostnames or IPs of all control plane hosts")
+	cmd.PersistentFlags().StringVar(&gatherBootstrapOpts.sshKey, "key", "", "Path to SSH key that should be used to connect with bootstrap host. If empty, SSH keys from user's environment will be used")
```

Comment:
Hopefully obvious from the context, but I'd rather spell this out as "Path to SSH private key...".

Reply:
fixed with https://github.com/openshift/installer/compare/16c8fee3bd84fb272d50a107abc544c624bc26d6...4326be6c46e8e43025b370c7b7de9ef41872868b